mon-pix/app/controllers/terms-of-service.js

```diff
@@ -24,7 +25,11 @@ export default class TermsOfServiceController extends Controller {
       if (this.session.attemptedTransition) {
         this.session.attemptedTransition.retry();
       } else {
-        this.transitionToRoute('profile');
+        if (ENV.APP.FT_DASHBOARD) {
+          this.transitionToRoute('user-dashboard');
+        } else {
+          this.transitionToRoute('profile');
+        }
```

Comment:
pour moi on a pas à faire ça ici, il n'y a que la page index, qui connait cette logique : ici on devrait seulement rediriger vers la page par défaut `this.transitionToRoute('');`

Reply:
by the way, j'avais vu ces petits trucs, je les avais inclus dans la PR #2482 ^^

Reply:
C'est pris avec le rebase :) Merci !